app/models/container.rb

```diff
@@ -2,6 +2,7 @@ class Container < ApplicationRecord
   include SupportsFeatureMixin
   include NewWithTypeStiMixin
   include ArchivedMixin
```

Comment:
@kbrock Not sure what you mean here. Do you mean to unify `ArchivedMIxin` and `ArchivedPurgingMixin`? I think that would work well